src/params.cpp

```diff
@@ -258,6 +258,11 @@ Params::Params(int argc, char* const argv[])
 		}
 #endif
 
+		if ((strcmp(argv[i], "--full-validation") == 0)) {
```

Comment:
I think you left extra brackets `((` and `))` after copy-pasting, you don't need them

Reply:
removed extra ( ) in https://github.com/SChernykh/p2pool/pull/374/commits/164e598677bdf7889328a5c766e81d50911abdee